src/kOS/Function/KAC.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
```

Comment:
There isnt a good pattern for this for you to follow but I would prefer this file in the KAC addon's folder.